lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -618,50 +588,12 @@ public function link_accounts_validation( $old_options, $input ) {
 		return $this->rule_validation( $old_options, $input, 'link_auth0_users', WP_Auth0_RulesLib::$link_accounts['name'] . '-' . get_auth0_curatedBlogName(), $link_script );
 	}
 
-	// TODO: Deprecate
+	/**
+	 * @deprecated 3.6.0 - The setting this validated, passwordless_method, has been removed
```

Comment:
> The setting this validated

typo?

Reply:
@lbalmaceda -No but maybe poorly worded. This validated the `passwordless_method` setting but the setting is being removed so the validation is going as well.